apps/emqx_durable_storage/src/emqx_ds.erl

```diff
@@ -30,7 +30,7 @@ It takes care of forwarding calls to the underlying DBMS.
     shard_of/2,
     list_shards/1,
     list_generations_with_lifetimes/1,
-    drop_generation/2,
+    drop_slab/2,
```

Comment:
Q: What about `add_generation/1` and `list_generations_with_lifetimes/1`?

Reply:
`add_generation` is a valid name, because it affects all shards. `list_generations_with_lifetimes` should be indeed changed. I'll do it in a follow-up.